ynput/ayon-core
# Workfiles tool: Folder change update tasks correctly

## Changelog Description
Fix edge cases when change of folder does not update tasks.

## Additional info
Tasks model is waiting until the tasks for the folder are fetched if are already fetched.

## Testing notes:
NOTE: This is edge case, usually happens on first open of workfile, and selecting differnet folder in different level.
1. Open workfiles tool.
2. Change of folder should propagate correct tasks in tasks view.
